styles/style.css

```diff
@@ -96,3 +96,153 @@ body {
     margin-top: 40px;
 }
 
+/* Styles for main section*/
+.main-container {
+    min-width: 1440px;
+    min-height: 100vh;
+    padding: 51px 104px 120px;
+    margin: 0 auto;
+    background: url(../img/project_images/mainpage-image.png) no-repeat center center;
```

Comment:
Done. Without these two properties the background looks bad on small screens. But no difference at large screens. Can you please explain? Nobody explained this.

Reply:
It's according to design for 1440px demension, if you do 1440px, you will see, it will be closer to design
For small screens you can use media queries in the future
you can read about property values here:
[background-size](https://www.w3schools.com/cssref/css3_pr_background-size.asp)
[background-position
](https://www.w3schools.com/cssref/pr_background-position.asp)